### DISCUSSION TITLE
🎒 Can't Verify Because of last name

### DISCUSSION THREAD

**Discussion by oliverchange** [community/community#86574] [GitHub Education]
### Hi everyone,

I submitted the request and get the email. It said I had to correct my billing information with my first name and last name exactly as it appears in my academic affiliation document. But my single legal last name is Chinese and only have 1 character. I can’t save my billing information because there an error said “Last name is too short."

![CleanShot 2024-01-11 at 16 58 04@2x](https://github.com/community/community/assets/126778947/25b411f9-2046-48a1-874d-0e37778dce6e)
![Screenshot_20240111_170108_Gmail](https://github.com/community/community/assets/126778947/c698a457-a581-48a9-a389-4cc2cf1a23b5)

**Comment by kmadaminjanova0508** [comment]
Hi try to write u r Last name with English letters :)

**Comment by faridnec** [comment]
Hey, if your last name is too short, maybe you can consider using a pseudonym or adding an additional character (e.g., a space) to meet the required length.

**Comment by Dong-Chen-1031** [comment]
I have the same problem. My last name is "陳", and I am also trying to get the GitHub Education benefits.
![image](https://github.com/user-attachments/assets/4f13508b-abaf-4ded-bada-873212259f8b)
![image](https://github.com/user-attachments/assets/3831433b-cb17-4dc0-8a64-3c3cb37c993c)

**Comment by samus-aran** [comment]
Thanks for this! GitHub Education can't assist on this particular element of the platform, but I can suggest that in order to proceed with the SDP application that you do translate your name and evidence in order to meet the minimum number of characters 😅 

I've included a section from our [Student Developer Pack Application + Copilot Pro FAQs](https://github.com/orgs/community/discussions/111352) discussion I feel would best assist.

- [I need to translate my evidence. What can I do?](https://github.com/orgs/community/discussions/111352#translate-my-evidence)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
